crates/ironrdp-server/src/handler.rs

```diff
@@ -0,0 +1,140 @@
+use ironrdp_pdu::input::{
+    fast_path, mouse::PointerFlags, mouse_x::PointerXFlags, scan_code, unicode, MousePdu, MouseXPdu,
+};
+
+/// Keyboard Event
+///
+/// Describes a keyboard event received from the client
+///
```

Comment:
Personally, I think enums are a more natural way to represent these than the bitflags found in the pdus. And this way we can also combine both FastPath and SlowPath events into a single type.
What do you think @CBenoit?

Reply:
I’m not sure we’ll ever have to deal with Slow-Path packets, but I do agree. We have [something similar for client side in the `ironrdp-input` crate](https://github.com/Devolutions/IronRDP/blob/55176ff99b8c72076c49ec4a2333ba9324798bc5/crates/ironrdp-input/src/lib.rs#L129-L136) because it gives [code easier to read and write](https://github.com/Devolutions/IronRDP/blob/55176ff99b8c72076c49ec4a2333ba9324798bc5/crates/ironrdp-client/src/gui.rs#L79-L88) (imo), while [dealing with the raw RDP PDUs is not super fun](https://github.com/Devolutions/IronRDP/blob/55176ff99b8c72076c49ec4a2333ba9324798bc5/crates/ironrdp-input/src/lib.rs#L262-L292). What is missing in your opinion? @elmarco

Reply:
I am afraid the server implementation may have issues with some simpler abstraction if we lack details like flags EXTENDED, or atomic DOWN|RELEASE. But there is a clear benefit of a common type for Slow/Fast events.

Reply:
Right, I will add the EXTENDED flag as a bool similar to [Scancode](https://github.com/Devolutions/IronRDP/blob/55176ff99b8c72076c49ec4a2333ba9324798bc5/crates/ironrdp-input/src/lib.rs#L64) from `ironrdp-input`

Reply:
> I am afraid the server implementation may have issues with some simpler abstraction if we lack details like flags EXTENDED, or atomic DOWN|RELEASE. 

Indeed. Probably we can get started with the current approach and revisit later if we stumble upon any problem because of this. Maybe at some point in the future we'll want to unify with `ironrdp-input` and make sure the abstraction is still reflecting the original input accurately if anything is missing.